src/pages/Lend/LendByCategoryPage.vue

```diff
@@ -413,10 +414,8 @@ export default {
 
 		if (this.categoryDescriptionExperimentVersion === 'variant-a') {
 			this.showCategoryDescription = false;
-			this.$kvTrackEvent('Lending', 'EXP-CASH-658-Mar2019', 'a');
 		} else if (this.categoryDescriptionExperimentVersion === 'variant-b') {
 			this.showCategoryDescription = true;
-			this.$kvTrackEvent('Lending', 'EXP-CASH-658-Mar2019', 'b');
```

Comment:
I believe these need to remain. What you've added to the template are click events on the paragraph of copy.